sinks_test.go

```diff
@@ -84,6 +89,30 @@ func TestZeroPipeSinks(t *testing.T) {
 	doSinksOnPipe(t, &Pipe{}, "zero")
 }
 
+func TestSHA256Sums(t *testing.T) {
+	t.Parallel()
+
```

Comment:
I don't use any blank lines within functions in this project (I don't think), so for consistency of style, shall we remove them here?